### PR TITLE
Feature/risk calculation

### PR DESCRIPTION
**Pull request by ChristianKienle**

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController+DynamicTableViewModel.swift [hunk 1]
```diff
@@ -162,8 +162,7 @@ private extension DynamicCell {
 		.exposureDetectionCell(ReusableCellIdentifer.riskRefresh) { viewController, cell, _ in
 			let state = viewController.state
 			cell.backgroundColor = state.riskTintColor
-			let components = Calendar.current.dateComponents([.minute, .second], from: Date(), to: state.nextRefresh ?? Date())
-			cell.textLabel?.text = String(format: text, components.minute ?? 0, components.second ?? 0)
+			cell.textLabel?.text = "Aktualisierung alle 24 Stunden"
```
Comment (haosap) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController+DynamicTableViewModel.swift:165]: will be fixed with next PR together with label issues.

File: src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift [hunk 2]
```diff
@@ -268,70 +165,68 @@ extension HomeInteractor {
...
+				lastUpdateDate: nil,
+				detectionInterval: detectionInterval,
+				detectionMode: detectionMode,
+				manualExposureDetectionState: riskProvider.manualExposureDetectionState
 			)
 		case .inactive:
-			inactiveConfigurator = HomeInactiveRiskCellConfigurator(incativeType: .noCalculationPossible, lastInvestigation: "Geringes Risiko", lastUpdateDate: dateLastExposureDetection)
+			inactiveConfigurator = HomeInactiveRiskCellConfigurator(
+				incativeType: .noCalculationPossible,
+				lastInvestigation: "Geringes Risiko",
```
Comment (haosap) [inline src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift:196]: localization

File: src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskLevelCollectionViewCell.xib [hunk 3]
```diff
@@ -4,110 +4,106 @@
     <dependencies>
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskLevelCollectionViewCell.xib:4]: This overrode all dark mode adjustments. (Resolve using "mine"?). I can re-apply the change, if you like.

Comment (haosap) [reply src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/Risk/RiskLevelCollectionViewCell.xib:4]: sure just do it. not overrode it on purpose.

**Review by haosap — APPROVED** [review]

**Review by llanghammer-sap — APPROVED** [review]

**Comment by ChristianKienle** [comment]
🎉

**Pull request merged**